### ISSUE TITLE
Update Google Analytics Script

### ISSUE THREAD

**Issue by gonzalezsieira** [andriusvelykis/reflow-maven-skin#38]
The script given by Google Analytics to put in the webpaches has changed, now it is given in the following way:

```
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'PUT-HERE-YOUR-GOOGLE-ANALYTICS-ID', 'auto');
  ga('send', 'pageview');

</script>
```

The auto-generated by the label `<googleAnalyticsAccountId>` is different, maybe from previous versions of Google Analytics. Please update the decorator to avoid compatibility problems with newer versions of Analytics.

**Comment by andriusvelykis** [comment]
Thanks for reporting. Unfortunately, I don't have any time to spend on this in the immediate future - I am busy writing up my PhD thesis.

I will look into it (and update) later. I think as a workaround you can add the correct version of your script directly as HTML, right?

**Comment by gonzalezsieira** [comment]
Yes, that's right. But don't worry, the main features of Analytics work even with the old script. I think it is only needed to activate the new "Universal Analytics" which is being introduced recently.
